src/Model/Tag.php

```diff
@@ -339,6 +339,23 @@ public static function getByURIId(int $uri_id, array $type = [self::HASHTAG, sel
 		return DBA::selectToArray('tag-view', ['type', 'name', 'url'], $condition);
 	}
 
+	/**
+	 * Return a string with all tags and mentions
+	 *
+	 * @param integer $uri_id
+	 * @return string tags and mentions
+	 */
+	public static function tagTextFromItemId(int $uri_id)
```

Comment:
Please rename for clarity:
```suggestion
	public static function getCommaSeparatedStringByURIId(int $uri_id)
```

Reply:
You could possibly have called it `getACommaSeparatedStringOfHashtagsAndMentionsLikeTheOldFunctionByURIId` instead as well :-)

P.S.: I will apply the changes

Reply:
More words is better than less!

Reply:
We are having these discussions at work as well. :-)